src/components/ui/ProgressBarStep.js

```diff
@@ -0,0 +1,75 @@
+import { Check } from '@styled-icons/boxicons-regular'
+import React from 'react'
+import styled from 'styled-components/macro'
```

Comment:
There's been a couple usages of this in the past where it's been hard to avoid, but importing the theme is not good practice. You should access it inside the styled component. I see a couple usages here in PrimaryText where you can avoid this

Reply:
Importing the theme is not good practice because then it's not swappable